src/drt/src/pa/FlexPA_unique.cpp

```diff
@@ -278,6 +278,24 @@ bool UniqueInsts::hasUnique(frInst* inst) const
   return inst_to_unique_.find(inst) != inst_to_unique_.end();
 }
 
+void UniqueInsts::forceInstAsClassHead(frInst* target_inst)
+{
+  UniqueInsts::InstSet& unique_class = *inst_to_class_[target_inst];
+  UniqueInsts::InstSet temporary_set;
+  for (frInst* inst : unique_class) {
+    if (inst != target_inst) {
+      temporary_set.insert(inst);
+    }
+  }
+  for (frInst* inst : temporary_set) {
+    deleteInst(inst);
+  }
+  // target inst is now the only inst in the set, therefore its head
+  for (frInst* inst : temporary_set) {
+    addInst(inst);
+  }
+}
+
```

Comment:
I hate having unused debug code. DRT is already full of it and at some point, no one knows what it does. Could you either actually use it or keep it locally?
Also, better to have in a separate PR.